x/auth/mapper_test.go

```diff
@@ -113,11 +111,43 @@ func BenchmarkAccountMapperGetAccountFound(b *testing.B) {
 		arr := []byte{byte((i & 0xFF0000) >> 16), byte((i & 0xFF00) >> 8), byte(i & 0xFF)}
 		addr := sdk.AccAddress(arr)
 		acc := mapper.NewAccountWithAddress(ctx, addr)
+		acc.SetCoins(sdk.Coins{sdk.NewCoin("LTC", sdk.NewInt(1000)),
```

Comment:
I think it might be worthwhile moving the coins list out to a var outside of the benchmark loop.